docs/concepts/what-is-a-query.md

```diff
@@ -8,51 +8,37 @@ tags:
-  This means that Query handling logic can not spawn Activity Executions.
-
-In many SDKs the Temporal Client exposes a predefined `_stack_trace` Query that returns the stack trace of all the threads owned by that Workflow Execution.
-This is a great way to troubleshoot a Workflow Execution in production.
-
-### Stack Trace Query
-
-There is a built-in Query type named `__stack_trace`.
-If a Workflow Execution has been stuck at a state for longer than an expected period of time, you can send a Query to return the current call stack. The `__stack_trace` Query name does not require special handling in your Workflow code.
-
-### Consistent Query
-
-A Query can be one of two consistency levels, _eventual_ and _strong_.
-Consider if you were to send a Signal to a Workflow Execution with the intent to update its state, and then immediately send a Query to get the state.
+- [How to use Queries in Go](/docs/go/how-to-use-queries-in-go)
+- [How to use Queries in Java](/docs/java/queries)
+- [How to use Queries in PHP](/docs/php/queries)
+- [How to send a Query to a Workflow Execution using tctl](/docs/tctl/workflow/query)
 
-#### Eventual consistency
+Queries are sent from a Temporal Client to a Workflow Execution.
+The API call is synchronous.
+The Query is identified at both ends by a Query name.
+The Workflow must have a Query handler that is developed to handle that Query and provide data that represents the state of the Workflow Execution.
 
-The Query may or may not return the updated state that quickly.
-However, there is a guarantee that eventually the Query would return the actual state.
-This is what it means for a query to be eventually consistent.
+Queries are strongly consistent and are guaranteed to return the most recent state.
+This means that the data reflects the state of all confirmed Events that came in before the Query was sent.
+An Event is considered confirmed if the call creating the Event returned success.
+Events that are created while the Query is outstanding may or may not be reflected in the Workflow state the Query result is based on.
+Strongly consistent Queries may have a small amount of latency.
 
-#### Strong consistency
+A Query can carry arguments to specify which data it is requesting.
+And each Workflow can expose data to multiple types of Queries.
 
-Query has another consistency level called strong consistency.
-A strongly consistent Query is guaranteed to return the state, which includes all Events that came before the Query was issued.
-An Event is considered to have come before a Query if the call creating the Event returned success before the Query was issued.
-Events that are created while the Query is outstanding may or may not be reflected in the Workflow state the Query result is based on.
+A Query must never mutate the state of the Workflow Execution – that is, Queries are **read-only** and cannot contain any blocking code.
```

Comment:
```suggestion
A Query must never mutate the state of the Workflow Execution—that is, Queries are _read-only_ and cannot contain any blocking code.
```
Formatting change per "Italic" section of https://developers.google.com/style/text-formatting.